python/cudf/cudf/_lib/datetime.pyx

```diff
@@ -69,13 +69,13 @@ def ceil_datetime(Column col, object field):
             c_result = move(libcudf_datetime.ceil_day(col_view))
         elif field == "H":
             c_result = move(libcudf_datetime.ceil_hour(col_view))
-        elif field == "T":
+        elif field == "T" or field == "min":
```

Comment:
Who knows when pandas will add support for more equivalents. This notation looks a bit cleaner and will only get better if we need to add more options.
```suggestion
        elif field in {"T", "min"}:
```

Reply:
@vyasr This doesn't work in the `with nogil:` because you are checking containment in a Python object. See previous discussion here: https://github.com/rapidsai/cudf/pull/9571#discussion_r743821096

Reply:
I didn't even notice that, yeah this won't work without the GIL. @mayankanand007 feel free to ignore all of the suggestions like this then. You could do the cascading if-else statements with the GIL just to select the desired function, then call that inside a with nogil block, but that seems a large and potentially nontrivial change to get the function typedef just right.